astroquery/desi/tests/test_desi_remote.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import astroquery.desi
+
+from astropy.io.fits import HDUList
+from astropy.coordinates import SkyCoord
+from astropy.coordinates import Angle
+from astropy.table import Table
+from astroquery.exceptions import NoResultsWarning
+
+
+@pytest.mark.remote_data
+class TestLegacySurveyClass:
+
+    def test_query_region(self):
+
+        ra = Angle('11h04m27s', unit='hourangle').degree
```

Comment:
this is just a float, why the extra Angle, etc wrapping here and below? we don't need to test that `Angle` and `SkyCoord` work as expected here, that's an upstream task for astropy.